src/components/Activities/ActivitiesList.tsx

```diff
@@ -0,0 +1,65 @@
+import { Activities } from "./Definition";
+import ListView from "./ListView";
+
+const builtInActivities: Activities[] = [
```

Comment:
Please move this array out to `App` and pass it as an `activities` prop to this component

Reply:
Done in 6f5ba69